pkg/ds/unitset.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2023- IBM Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ds
+
+type UnitSet struct {
```

Comment:
`UnitSet` not used?

Reply:
No. I thought it could be used as the edge case of a dynamically-bounded product, but it didn't work.